src/acceptance/assets/app/go_app/deploy.sh

```diff
@@ -1,37 +1,53 @@
 #!/bin/bash
-
 set -euo pipefail
-config=${CONFIG:-}
+
+# read content of config right at the beginning to avoid errors after switching directories
+config="$(cat "${CONFIG:-}" 2> /dev/null || echo "")"
+
 function getConfItem(){
-  val=$(jq -r ".$1" "${config}")
+  val=$(jq -r ".$1" <<< "$config")
   if [ "$val" = "null" ]; then return 1; fi
   echo "$val"
 }
-if [ -z "${config}" ]; then
+
+if [ -z "$config" ]; then
   echo "ERROR: Please supply the config using CONFIG env variable"
   exit 1
 fi
 
-script_dir=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
-app_dir="$( realpath -e "${script_dir}/build")"
-service_name=$(getConfItem service_name)
-
-cf create-org test
-cf target -o test
-cf create-space test_app
-cf target -s test_app
-
-cf enable-service-access "${service_name}" -b "$(getConfItem service_broker)" -p  "$(getConfItem service_plan)" -o test
-cf create-service "${service_name}" "$(getConfItem service_plan)" "${service_name}" -b "$(getConfItem service_broker)" --wait
```

Comment:
unrelated: I removed the `{}`-surroundings of variables where it doesn't make sense to have them

Reply:
Oh, what a pity! :( They appear more readable. But perhaps that's developer-taste.

Reply:
I will bring them back after reading your comment and this here https://stackoverflow.com/a/8748880/1294742 😊